crates/kitsune_p2p/kitsune_p2p/src/spawn/actor/bootstrap.rs

```diff
@@ -105,8 +113,11 @@ fn local_now() -> crate::types::actor::KitsuneP2pResult<u64> {
 ///
 /// There is no input to the `now` endpoint, just `()` to be encoded as nil in messagepack.
 #[allow(dead_code)]
-pub async fn now(url: Option<Url2>) -> crate::types::actor::KitsuneP2pResult<u64> {
-    match do_api(url, OP_NOW, ()).await {
+pub async fn now(
+    url: Option<Url2>,
+    net: BootstrapNet,
```

Comment:
Correct. The url identifies the specific bootstrap server, but that one server can support multiple nets.

Reply:
I was wondering about the `?net={}` param I saw on other URLs

Reply:
Ah, okay. Yeah, i mean the net parameter does end up being a query string on the url, like `bootstrap.holo.host?net=tx5`. We just defer adding that until the end, because the url in the holochain config shouldn't have it yet, since they could switch back and forth which network they are using separately in the config.